lua/conform/runner.lua

```diff
@@ -48,6 +48,7 @@ M.build_cmd = function(formatter_name, ctx, config)
       :gsub("$FILENAME", ctx.filename)
       :gsub("$DIRNAME", ctx.dirname)
       :gsub("$RELATIVE_FILEPATH", compute_relative_filepath)
+      :gsub("$EXTENSION", ({ ctx.filename:match("(.*)(%..*)") })[2] or "")
```

Comment:
I think `ctx.filename:match(".*(%..*)$")[1]` would be simpler

Reply:
Wasn't sure if you were allowed to do that it's been awhile since I've wrote Lua. Thought the returned values were unpaired already.